catalyst_voices/packages/internal/catalyst_voices_models/analysis_options.yaml

```diff
@@ -1,6 +1,6 @@
 include: package:catalyst_analysis/analysis_options.yaml
 
 analyzer:
-  exclude: [build/**, lib/*.g.dart, lib/generated/**]
+  exclude: [ build/**, lib/**.g.dart, lib/generated/** ]
```

Comment:
on my IDE formatting .yaml doesn't add these empty spaces here, you're sure this is the right format?

Reply:
Yep, lets revert, otherwise we will be overriding each other's changes.